### PR TITLE
More info (for real this time)

### PR DESCRIPTION
**Pull request by CATboardBETA**

### CODE REVIEW

**Review by hyperswine — COMMENTED** [review]
works for me, but doesnt pass tests

**Review by hyperswine — APPROVED** [review]
Looks good

File: quantii/src/lib.rs [hunk 1]
```diff
@@ -24,8 +24,12 @@
 
 extern crate novuskinc;
 // Missing intrinsics patch.
+extern crate alloc;
 extern crate compiler_builtins_patch;
```
Comment (AldaronLau) [inline quantii/src/lib.rs]: The comment should apply to the line below.  Basically, `extern crate alloc` should be moved one line up.

File: quantii/src/lib.rs [hunk 2]
```diff
@@ -70,9 +74,31 @@ impl ardaku::System for System {
...
+            }
+            ArdakuError::Crash(c) => {
+                msg = "Ardaku: Error: Crash:".to_owned();
+                msg += &c.to_string();
+            }
+            ArdakuError::MissingMemory => {
+                msg = "Ardaku: Error: Ran out of memory".to_owned()
+            }
+        },
+    }
```
Comment (AldaronLau) [inline quantii/src/lib.rs]: There's no need to use allocation here.  Since all of the messages are static `&str` types.

File: quantii/src/lib.rs [hunk 3]
```diff
@@ -70,9 +74,31 @@ impl ardaku::System for System {
...
+    System.write(b"Quantii OS v0.0.1");
+
+    System.write(b"\n=== ARDAKU STARTED ===\n");
+
+    let mut msg: String;
+    match ardaku::start(System, APP_EXE) {
+        Ok(_) => msg = "\n=== ARDAKU SUCCESSFULLY EXECUTED ===\n".to_owned(),
+        Err(e) => match e {
+            ArdakuError::InvalidWasm => {
+                msg = "Ardaku: Error: Invalid WASM file".to_owned()
```
Comment (AldaronLau) [inline quantii/src/lib.rs]: Should combine let and match into one expression.

```suggestion
    let msg = match ardaku::start(System, APP_EXE) {
        Ok(_) => "\n=== ARDAKU SUCCESSFULLY EXECUTED ===\n",
        Err(e) => match e {
            ArdakuError::InvalidWasm => {
                "Ardaku: Error: Invalid WASM file"
```

**Review by AldaronLau — CHANGES_REQUESTED** [review]
What's the initfs/home/.qiishenv file for?

File: config/linux.toml [hunk 4]
```diff
@@ -26,3 +22,7 @@ rpath = false
 
 [build]
 target = "aarch64-unknown-none"
+
+[unstable]
+build-std = ["core", "alloc", "compiler_builtins"]
+#build-std-features = ["compiler-builtins-mem"]
```
Comment (AldaronLau) [inline config/linux.toml]: We'll want to undo all changes to this file.

**Review by AldaronLau — CHANGES_REQUESTED** [review]

**Review by AldaronLau — APPROVED** [review]

**Pull request merged**